starter-code/index.html

```diff
@@ -3,40 +3,100 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/css_reset.css">
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+<!-- Header -->
+<header>
+	<div class="logo">
+		<img src="./images/spotify-logo.png" alt="logo">
+	</div>
+
+	<ul>
+		<li>Premium</li>
+		<li>Discover</li>
+		<li>Help</li>
+		<li>Download</li>
+	</ul>
+   
+</header>
+
+<!-- First section -->
+
+<section>
+	<h1>Music for everyone.</h1>
+	<h4>
+		Spotify is now free on mobile, tablet and computer.
+		<br>
```

Comment:
Like I said to others, br tags should only be used for poems, addresses, or other cases where the line break is part of language conventions; line break for styling purposes are better dealt with from the css.
By using flexbox and managing the width of your box, you should be able to have the line break where you intend to (you may have to use the flex-wrap property). You can also wrap the second line in a span and add a display: block to this span in the css!